.github/workflows/benchmark.yml

```diff
@@ -1,8 +1,7 @@
 on:
   push:
     branches:
-      - master
-      - main
```

Comment:
was removing main intentional?
same question for the other workflows

Reply:
no thx